lex/bot/OscarBot.json

```diff
@@ -123,7 +123,7 @@
         "maxAttempts": 5,
         "messages": [
             {
-                "content": "Sorry, can you please repeat that?",
+                "content": "Dang, I couldn't understand that. Ask 'What can I ask you?' for phrases I will understand.",
```

Comment:
This is the prompt when they fail to elicit a slot, so they will not be able to ask 'what can I ask you' in this case. It's more 'can you check for typos and type it again?'